### PR TITLE
feat(pages): new guides

### PR DESCRIPTION
**Pull request by rishi-raj-jain**
In this PR:

- new guides page with HTML created from Markdown files

cc @ruf-io

### CODE REVIEW

**Comment by rishi-raj-jain** [comment]
Can I get the logs why this failed?

File: .env.example [hunk 1]
```diff
@@ -11,6 +11,7 @@ NEXT_PUBLIC_ALGOLIA_INDEX_NAME=
 NEXT_PUBLIC_ALGOLIA_BLOG_INDEX_NAME=
 
 NEXT_PUBLIC_DOCS_GITHUB_PATH=https://github.com/neondatabase/website/tree/main/content/docs/
+NEXT_PUBLIC_GUIDES_GITHUB_PATH=https://github.com/neondatabase/website/tree/main/content/guides/
```
Comment (rishi-raj-jain) [inline .env.example:14]: I think this went missing from the deployed branch on Vercel?

**Comment by danieltprice** [comment]
Hey @rishi-raj-jain 

Looks like a path issue. The referenced font is under public, so perhaps it's just the number of directory levels.
We'll have to ask PixelPoint (our website team) to give you Vercel access so that you can access the logs and Vercel previews.

image

**Comment by rishi-raj-jain** [comment]
> Hey @rishi-raj-jain
> 
> Looks like a path issue. The referenced font is under public, so perhaps it's just the number of directory levels. We'll have to ask PixelPoint (our website team) to give you Vercel access so that you can access the logs and Vercel previews.
> 
> image

Thank you! I've updated the PR.

**Comment by k8isdead** [comment]
@rishi-raj-jain Hey, I've sent you the Vercel invite to [rishi18304@iiitd.ac.in](mailto:rishi18304@iiitd.ac.in)

**Comment by k8isdead** [comment]
@danieltprice @bgrenon @ruf-io You can also approve the deployments for the contributor's commits here is the PR

**Comment by rishi-raj-jain** [comment]
> @rishi-raj-jain Hey, I've sent you the Vercel invite to [rishi18304@iiitd.ac.in](mailto:rishi18304@iiitd.ac.in)

Thanks!

**Comment by k8isdead** [comment]
@rishi-raj-jain Hi Rishi, we have finalized the design for the Guides page and the Individual guide pages, we are working on it in the PR #1671, please do not make any content changes until we finish the preview and instructions to review it for you and @ruf-io in order to prevent context loss between branches. We will be able to introduce any feedback and content changes before going live with the pages 🙌🏻

**Comment by rishi-raj-jain** [comment]
@k8isdead 

Sure thing.

Thank you!

**Comment by rishi-raj-jain** [comment]
Closed against #1671.

**Pull request closed without merging**